sdk/identity/Azure.Identity/src/Credentials/AzurePowerShellCredential.cs

```diff
@@ -252,10 +252,26 @@ private void GetFileNameAndArguments(string resource, string tenantId, out strin
     Write-Output '{AzurePowerShellNoAzAccountModule}'
     exit
 }}
+$tenantId = '{tenantIdArg}'
+$params = @{{ ResourceUrl = '{resource}', 'WarningAction' = 'Ignore' }}
+
+if ($tenantId.Length ge 0) {{
```

Comment:
```suggestion
if ($tenantId.Length -gt 0) {{
```

Reply:
good catch - I had fixed this in some external test code and must have missed updating it.